### PR TITLE
feat: add SCSI bootstrap script for datasets

### PR DESCRIPTION
**Pull request by jywadhwani**
## Checklist
- [X] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by mars-lan — COMMENTED** [review]
A better location might be https://github.com/linkedin/datahub-gma/tree/master/dao-impl/ebean-dao

File: gms/database/scripts/index/dataset-bootstrap.sql [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  FROM   (SELECT DISTINCT urn,
+                          length(urn) AS urnLength
+          FROM   metadata_aspect
+          WHERE  urn NOT IN (SELECT DISTINCT urn
+                             FROM   metadata_index
+                             WHERE  aspect="com.linkedin.common.urn.DatasetUrn")
+                 AND urn RLIKE "^urn:li:dataset:\\(urn:li:dataPlatform:[^:,]+,[^:,]+,[^:,]+\\)$"
+                 AND aspect="com.linkedin.common.Status"
+                 AND version=0
+                 AND metadata LIKE '%false%')
```
Comment (nagarjunakanamarlapudi) [inline gms/database/scripts/index/dataset-bootstrap.sql:25]: + @keremsahin1 
Do we still want to exclude deleted datasets not to make into SCSI ?  I couldn't recollect our earlier meeting's conclusion on this.

Comment (keremsahin1) [reply gms/database/scripts/index/dataset-bootstrap.sql:25]: We decided to index everything regardless of removal status into the SCSI. I need to change internal bootstrap script.

**Comment by keremsahin1** [comment]
> A better location might be https://github.com/linkedin/datahub-gma/tree/master/dao-impl/ebean-dao

This script is entity specific. I don't think we would want to move entity specific stuff into `datahub-gma`.

**Comment by mars-lan** [comment]
> > A better location might be https://github.com/linkedin/datahub-gma/tree/master/dao-impl/ebean-dao
> 
> This script is entity specific. I don't think we would want to move entity specific stuff into `datahub-gma`.

Argh that makes sense. nvm.

File: gms/database/scripts/index/dataset-bootstrap.sql (hunk 1 above)
Comment (jywadhwani) [reply gms/database/scripts/index/dataset-bootstrap.sql]: Updated the script to index all urns irrespective of their `removed` flag.

**Review by keremsahin1 — APPROVED** [review]

**Pull request merged**